backend/src/utils/resourceUtils.ts

```diff
@@ -645,12 +645,17 @@ export const cleanupGPU = async (fastify: KubeFastifyInstance): Promise<void> =>
     .readNamespacedConfigMap(CONFIG_MAP_NAME, fastify.kube.namespace)
     .then(() => {
       // Found configmap, not continuing
+      fastify.log.info(`GPU migration already completed, skipping`);
       return false;
     })
     .catch((e) => {
       if (e.statusCode === 404) {
         // No config saying we have already migrated gpus, continue
         return true;
+      } else {
+        throw `fetching gpu migration configmap had a ${e.statusCode} error: ${
+          e.response?.body?.message ?? e?.response?.statusMessage
```

Comment:
Same here, I like this more, but just for the sake of consistency we might want to decide wether to use one or another

Reply:
i like using `??` more